packages/venia-ui/lib/components/CartPage/ProductListing/product.css

```diff
@@ -85,16 +84,21 @@
     grid-area: stock;
 }
 
+.stockStatusMessage:empty {
+    display: none;
+}
+
 @media (max-width: 640px) {
     .details {
-        grid-template-areas:
-            'name'
-            'options'
-            'price'
-            'stock'
-            'quantity';
-        grid-template-columns: auto;
-        grid-template-rows: auto;
+        grid-template: none;
+    }
+
+    .name,
+    .options,
+    .price,
+    .stockStatusMessage,
+    .quantity {
+        grid-area: auto;
```

Comment:
This resets `grid-area` for each grid item, so each one creates a new row.

Reply:
Reset back to default, I like it. Do we have a strong preference to avoid named template areas?

Reply:
Yes, we prefer sensible implicit grids over explicit grids in general, since they more readily accommodate new (unanticipated) grid items.